### DISCUSSION TITLE
Full-width image problem in editor

### DISCUSSION THREAD

**Discussion by kyledenlinger** [knowledgefutures/pubpub#2240] [Support]
When creating a new pub, if you insert a full-width image before adding any text, what results is a pub with a solo image that fills the entire editor and doesn't allow one to focus the cursor to add a line before or after the image. The same might also be true for other media embeds. I have been able to work around this by lowering the image width, adding text, then editing the image again, or selecting the block quote format, which allows me to then focus the cursor.

**Comment by kyledenlinger** [comment]
Possibly related: [iframe problems in editor](https://github.com/pubpub/pubpub/discussions/2239)

**Comment by gabestein** [comment]
Hi Kyle -- sorry you're having the issue. Can you let us know what browser and operating system you'r working with? Fwiw, for me on Chrome / Mac OS, I can access the cursor by clicking outside the image. This isn't super intuitive and could be improved, but at least provides an easier workaround.

**Comment by kyledenlinger** [comment]
Chrome on Mac. To be clear, I am also able to get the cursor by clicking
outside of the image when the image is not full width. If it is full width,
however, and there's nothing else previously added to the text editor, the
image effectively fills the editor and leaves no space for me to click
into. If it'd be possible to tab into the editor and arrow the cursor to
the end of the image, that would help, but I can't seem to do that. It
might help to automatically include a line break after each image, which a
user could delete, but at least there'd be space there to click into.

Sorry, this is super picky and easily worked around by changing the image
alignment. Just thought I'd point it out.

On Thu, Sep 8, 2022 at 10:44 AM Gabriel Stein ***@***.***>
wrote:

> Hi Kyle -- sorry you're having the issue. Can you let us know what browser
> and operating system you'r working with? Fwiw, for me on Chrome / Mac OS, I
> can access the cursor by clicking outside the image. This isn't super
> intuitive and could be improved, but at least provides an easier workaround.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pubpub/pubpub/discussions/2240#discussioncomment-3595548>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGWEYG22NKSI63A3UT3HH7LV5H3WBANCNFSM6AAAAAAQHZ67UU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Kyle Denlinger he/him/his

*Digital Pedagogy & Open Education Librarian*


Wake Forest University, Z. Smith Reynolds Library <https://zsr.wfu.edu/>

***@***.*** | meet with me > <https://calendly.com/kdenlinger>

**Reply by gabestein** [reply]
Not at all! We appreciate the feedback. I didn't realize the issue was with full-width. My apologies for not reading more thoroughly. We should definitely have a better way to get back to the cursor in that scenario.
